editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -533,12 +569,28 @@ void SpriteFramesEditor::_prepare_sprite_sheet(const String &p_file) {
 		split_sheet_sep_y->set_value(0);
 		split_sheet_offset_x->set_value(0);
 		split_sheet_offset_y->set_value(0);
-		updating_split_settings = false;
+	} else {
+		dominant_param = int(EditorSettings::get_singleton()->get_project_metadata("sprite_frames", "dominant_param", -1));
+		if (dominant_param == PARAM_SIZE) {
+			split_sheet_size_x->set_value(split_data["dom_x"]);
+			split_sheet_size_y->set_value(split_data["dom_y"]);
```

Comment:
Need to compute the other values here

Reply:
sorry didnt understood, the sep/offset is below outside the if statement, you mean compute them before? the dominants?

Reply:
also I have found a problem, if you close the project and open again, the dominant values are set, but the others, lets suppose dominant is `hor/vert` then `size x/y` is saved, and when open the project the `size x/y` is `set_value` but `hor/vert` is not calculated, didnt understood why!

Reply:
See here:
https://github.com/godotengine/godot/blob/7df393312fa8eab98231f2c335366a1ef6205367/editor/plugins/sprite_frames_editor_plugin.cpp#L446-L460

and when open the project the size x/y is set_value but hor/vert is not calculated, didnt understood why!

The min/max of the sep/offset values are updated here and should be done so here as well IMO

> and when open the project the size x/y is set_value but hor/vert is not calculated, didnt understood why!

This is exactly what I'm talking about, they need to be calculated, see the code I referenced for how it is done